src/AlternateLife.RageMP.Net/Interfaces/ITypeParser.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace AlternateLife.RageMP.Net.Interfaces
+{
+    public interface ITypeParser
+    {
+        bool TryParse(string s, Type targetType, out object value);
```

Comment:
`s` is not a good name for an interface variable. Rename it to `inputString`, for example.